packages/react/package.json

```diff
@@ -64,9 +64,11 @@
     "@radix-ui/react-dropdown-menu": "0.1.6",
     "@radix-ui/react-slider": "0.1.4",
     "@radix-ui/react-tabs": "0.1.5",
+    "@types/dompurify": "2.3.3",
     "@xstate/react": "1.6.3",
     "classnames": "2.3.1",
     "deepmerge": "4.2.2",
+    "isomorphic-dompurify": "0.18.0",
```

Comment:
Importing the isomorphic version of dompurify in order to get the e2e and docs tests to pass. Isomorphic dompurify is basically a wrapper around dompurify that allows it to run on the server side as well as the client.

Reply:
I'm nervous about depending on this package which hasn't been battle tested as much as the original. I'd prefer that we create our own universal isomorphic wrapper around DOMPurify.